### ISSUE TITLE
Error on zinit install

### ISSUE THREAD

**Issue by thanh125643** [marlonrichert/zsh-autocomplete#339]
* `zsh-autocomplete` version: b29b16cd5f358833515707ae220d3ef573cad06c
* Zsh version: /usr/bin/zsh ubuntu/5.8-3ubuntu1
* Framework: none
* Plugin manager: zinit
* Operating system: linux-gnu Ubuntu20.04
   ```zsh
    (anon):21: bad pattern: funcs=(\n      ~zsh-autocomplete/functions/.autocomplete.*~*.zwc(N-.:a)\n  )
    ```
install zsh-autocomplete in zinit make this weird error. Old version run normal

**Comment by marlonrichert** [comment]
> install zsh-autocomplete in zinit make this weird error. Old version run normal

Then that's a bug in `zinit`. Please report it there.

**Comment by yutkat** [comment]
I think this problem seems to have been fixed from aab02776676f2a584cf843edfa7eb9dcb0e39e60

**Comment by marlonrichert** [comment]
Yes, but the problem 
```zsh
 (anon):21: bad pattern: funcs=(\n      ~zsh-autocomplete/functions/.autocomplete.*~*.zwc(N-.:a)\n  )
```
which was the original problem of this bug report is still a bug in `zinit`.

**Comment by marlonrichert** [comment]
> I think this problem(`array parameter match set in enclosing scope in function ...`) seems to have been fixed from aab02776676f2a584cf843edfa7eb9dcb0e39e60

Actually, it was fixed in 0d930658ed410c6b24bf58af33d8dba28ee58c5d. I forgot to mention it. 🙂

**Comment by marlonrichert** [comment]
@thanh125643 PS: If you enjoy using my software, please consider [sponsoring me](https://github.com/sponsors/marlonrichert). 🙂
